start-site/src/main/resources/application.yml

```diff
@@ -1086,14 +1094,16 @@ initializr:
       content:
         - name: Zuul [Maintenance]
           id: cloud-zuul
-          description: Intelligent and programmable routing with Spring Cloud Netflix Zuul.
+          description: "Intelligent and programmable routing with Spring Cloud Netflix Zuul. Maintenance mode, suggested replacement: Spring Cloud Gateway."
           groupId: org.springframework.cloud
           artifactId: spring-cloud-starter-netflix-zuul
           compatibilityRange: "[2.0.0.RELEASE,2.4.0-M1)"
           links:
             - rel: guide
               href: https://spring.io/guides/gs/routing-and-filtering/
               description: Routing and Filtering
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-netflix/docs/2.2.6.RELEASE/reference/html/#router-and-filter-zuul
```

Comment:
Isn't there a chance that this link will go out of sync? It's tied to a particular version.

Reply:
This module is no longer available in `3.0.0`, so we can't use `current`.

Reply:
It might be better if we link to `https://docs.spring.io/spring-cloud-netflix/docs/2.2.x/reference/html/#router-and-filter-zuul`.